### DISCUSSION TITLE
How to choose T1 MRI

### DISCUSSION THREAD

**Discussion by momo-1120** [simnibs/simnibs#466] [6. Getting started]
Hi, I just started learning simnibs, and I want to generate a head model myself, but I'm not quite sure which T1 and T2 MRI scans to choose. In the HCP dataset, there are three types of files:
1. This package contains structural data preprocessed with the HCP structural pipeline, including MSM-Sulc and MSM-All registered versions of PreFreeSurfer, FreeSurfer, and PostFreeSurfer pipeline outputs.
2. This package includes MSM-Sulc and MSM-All registered versions of the FreeSurfer v5.3.0-HCP output directory created by the HCP Structural pipeline. These are intermediate files that are not in NIFTI/GIFTI/CIFTI format. Users wishing to run the Diffusion or Functional Preprocessing pipelines themselves can either (1) run the Structural pipeline first to generate the full output, or (2) add this package to the Structural Preprocessed package.
3. This package contains the NIFTI-formatted T1 and T2 weighted scans, the associated scans necessary to construct field maps, and a .csv containing scan acquisition details. The following auxiliary files are included: AFI, BIAS_BC, BIAS_32CH, FieldMap_Magnitude, and FieldMap_Phase.
Which one should I choose?

**Comment by TorgeW** [comment]
SimNIBS uses T1 and T2 weighted MRI scans (3) to create head models, please see the tutorial on that:
https://simnibs.github.io/simnibs/build/html/tutorial/head_meshing.html
